docs/src/terminology.md

```diff
@@ -22,23 +22,23 @@ A contiguous set of [entries](terminology.md#entry) on the ledger covered by a [
 
 ## blockhash
 
-A preimage resistant [hash](terminology.md#hash) of the [ledger](terminology.md#ledger) at a given [block height](terminology.md#block-height). Taken from the last [entry id](terminology.md#entry-id) in the slot
+A unique value ([hash](terminology.md#hash)) that identifies a record (block).  Solana computes a blockhash from the last [entry id](terminology.md#entry-id) of the block.
 
 ## block height
 
 The number of [blocks](terminology.md#block) beneath the current block. The first block after the [genesis block](terminology.md#genesis-block) has height one.
 
 ## bootstrap validator
 
-The first [validator](terminology.md#validator) to produce a [block](terminology.md#block).
+The server node ([validator](terminology.md#validator)) that produces the genesis (first) [block](terminology.md#block) of a block chain.
 
 ## CBC block
 
-Smallest encrypted chunk of ledger, an encrypted ledger segment would be made of many CBC blocks. `ledger_segment_size / cbc_block_size` to be exact.
+Thee smallest encrypted chunk of ledger, an encrypted ledger segment would be made of many CBC blocks. `ledger_segment_size / cbc_block_size` to be exact.
```

Comment:
typo

Reply:
whoops...tx

Reply:
hey Jon...was wondering if u could confirm with me that the term 'instruction' in Solana refers solely/commonly to code that a transaction uses to interface with a deployed program (smart contract).    In other words, we should try to refrain from using the term 'instruction' when referring to code in a smart contract.

If so, as such, the existing definition below is incorrect because it infers that a CPI can be made from a transaction whereas in fact CPI calls are only made from other smart contracts, not transactions.

I'll push through what i think is better here but please advise.

## instruction

The smallest unit of a [program](terminology.md#program) that a [client](terminology.md#client) can include in a [transaction](terminology.md#transaction). Within its processing code, an instruction may contain one or more [cross-program invocations](terminology.md#cross-program-invocation).

Reply:
The description in the definition is correct.
An instruction is the interface used to call programs both from transactions and to make CPI calls.
An instruction contains as its data the program it wishes to call, the accounts it wants to read or modify, and a data field that the program can use as inputs.

Reply:
thanks Jon...i will update this to reflect that an instruction as you say is " the interface used to call programs"

Please note that this also corrects the current documentation which infers that instructions exist solely in the realm of a transaction...excerpt from existing definition: "The smallest unit of a program that a client can include in a transaction"